app/assets/stylesheets/alchemy/image_library.scss

```diff
@@ -66,17 +120,39 @@ $image-overlay-form-width: 350px;
   }
 }
 
-#zoomed_picture_background {
+.picture-overlay-handle {
+  width: $picture-overlay-handle-width;
+  height: 100%;
+  right: $image-overlay-form-width;
+  cursor: pointer;
+
+  .icon-angle-double-right {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    @include transform(translate(-50%, -50%));
+    font-size: 14px;
+    color: $text-color;
+    @include transition(transform $image-overlay-transition-duration $image-overlay-transition-easing);
+  }
+
+  &:hover {
+    background-color: darken($medium-gray, 5%);
+  }
+}
+
+.zoomed-picture-background {
   @include box-sizing(border-box);
   width: 100%;
   height: 100%;
   padding-top: 2*$default-padding;
-  padding-right: $image-overlay-form-width + 2*$default-padding;
+  padding-right: $image-overlay-form-width + 2*$default-padding + $picture-overlay-handle-width;
```

Comment:
`2*$default-padding` should be written with a single space on each side of the operator: `2 * $default-padding`